### PR TITLE
Support for nested routing modules

### PR DESCRIPTION
**Pull request by dafo**
Support nested routing modules.
Support importing exported classes.

### CODE REVIEW

File: packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { App, FS_TOKEN, IFileSystem, TypeScriptUtils, Util } from "@igniteui/cli-core";
```
Comment (wnvko) [inline packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts:1]: in the end entire `TypeScriptFileUpdate` is mostly the same class exported in `cli.core`. I believe this was added here when POC was created. It is now time to remove this class and use the one from `core`. The changes added in this class should be transferred to the one in the `core`.

File: packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts [hunk 2]
```diff
@@ -33,8 +33,15 @@ export class TypeScriptFileUpdate {
 		this.initState();
 	}
 
-	public addRoute(filePath: string, linkPath: string, linkText: string, routesVariable = DEFAULT_ROUTES_VARIABLE) {
-		this.addRouteModuleEntry(filePath, linkPath, linkText, routesVariable);
+	public addRoute(
```
Comment (wnvko) [inline packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts:36]: This method is actually doing two things, at least. It adds routes as well as it adds imports. We should try to split this into two separate methods. Note, to do this we may need to create separate visitor for imports and routes, and all this should be done in `core` `TypeScriptFileUpdate`.

File: packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts (hunk 1 above)
Comment (dafo) [reply packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts:1]: There is an issue about that - https://github.com/IgniteUI/igniteui-cli/issues/983

File: packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts (hunk 2 above)
Comment (dafo) [reply packages/cli/templates/webcomponents/TypeScriptFileUpdate.ts:36]: I added this to the requirements in issue https://github.com/IgniteUI/igniteui-cli/issues/983

**Pull request merged**